File: Components/RewriteMe.Domain/Transcription/TranscribeItem.cs
using System;
using System.Collections.Generic;

namespace RewriteMe.Domain.Transcription
{
    public class TranscribeItem
    {
        public Guid Id { get; set; }

        public Guid FileItemId { get; set; }

        public Guid ApplicationId { get; set; }

        public IEnumerable<RecognitionAlternative> Alternatives { get; set; }

        public string UserTranscript { get; set; }

        public string SourceFileName { get; set; }

        public TimeSpan StartTime { get; set; }

        public TimeSpan EndTime { get; set; }

        public TimeSpan TotalTime { get; set; }

        public DateTime DateCreatedUtc { get; set; }

        public DateTime DateUpdatedUtc { get; set; }
    }
}

File: WebClient/src/app/_models/transcribe-item.ts
import { RecognitionAlternative } from './recognition-alternative';

export class TranscribeItem {
    public id: string;
    public transcript: string;
    public userTranscript: string;
    public alternatives: RecognitionAlternative[];
    public source: any;
    public startTime: string;
    public endTime: string;
    public totalTime: string;
    public dateCreated: Date;
}
